src/components/Pages/Service.js

```diff
@@ -12,7 +12,6 @@ import HtmlFromAdmin from 'components/HtmlFromAdmin';
 import ApplicationBlock from 'components/ApplicationBlock';
 import ContactDetails from 'components/Contact/ContactDetails';
 import SectionHeader from 'components/SectionHeader';
-import TileGroup from 'components/Tiles/TileGroup';
```

Comment:
thank you

Reply:
:-) there's  prob a few more of those out I think